salt/beacons/log.py

```diff
@@ -80,6 +80,20 @@ def beacon(config):
                   <tag>:
                     regex: <pattern>
     '''
+    '''
```

Comment:
Did you mean to replace the doc string above with this new one? As this sits presently, there are 2 docstring sections stacked one on top of the other. :)

Reply:
No, I wanted to add a more clear example, I can make it all into 1 doc string if that is preferred.